packages/cactus-plugin-ledger-connector-ethereum/src/test/typescript/integration/geth-contract-deploy-and-invoke-using-keychain-v1.test.ts

```diff
@@ -437,11 +443,11 @@ describe("Ethereum contract deploy and invoke using keychain tests", () => {
           secret: testEthAccount.privateKey,
           type: Web3SigningCredentialType.PrivateKeyHex,
         },
-      });
-      fail("Expected invokeContractV1 call to fail but it succeeded.");
-    } catch (error) {
-      expect(error).toBeTruthy();
-    }
+      })
+    ).rejects.toThrow();
+    
+    console.log("invokeContractV1 failed as expected");
```

Comment:
@ashnashahgrover Please remove the console.logs (applies to everywhere in the PR) and instead instantiate a logger object needed (I think it's OK to not log at all and just remove the console.log statements). Jest's own logging and assertions should contain enouhg information to debug a failing test.

Reply:
Replaced all console.log statements with logger.info statements